content/en/news/releases/1.24.x/announcing-1.24.4/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Announcing Istio 1.24.4
+linktitle: 1.24.4
+subtitle: Patch Release
+description: Istio 1.24.3 patch release.
+publishdate: 2025-03-12
```

Comment:
```suggestion
publishdate: 2025-03-17
```

This is hopeful. It's end of workday for me, no response about test results yet.